### ISSUE TITLE
Inline script commands in package.json are ignored

### ISSUE THREAD

**Issue by jakeleventhal** [depcheck/depcheck#672]
Reopening #584 and #257 

When a dependency is only used inside parenthesis of another command in package.json scripts, depcheck marks this dependency as unused. e.g.:
```
  "scripts": {
    "start": "concurrently \"cross-env BROWSER=none yarn browser\" \"wait-on http://localhost:3000 && electron .\""
  },
  "devDependencies": {
    "concurrently": "^3.5.1",
    "cross-env": "^5.1.6",
    "wait-on": "^2.1.0"
  }
```
Here, cross-env and wait-on are marked as unused.

This problem also happens with husky and jest

**Comment by TomNUSDS** [comment]
Came here to report this very same issue. 
Something as simple as this generates a glaring false positive report.

```json
  "scripts": {
    "build": "sass src/global.scss:src/content/generated/global.out.css"
  },
  "devDependencies": {
    "sass": "^1.47.2"
  },
```

**Comment by dobesv** [comment]
If you look at https://github.com/depcheck/depcheck/blob/main/src/special/bin.js you can see the rules it uses - they aren't comprehensive.  You could try opening a PR to try and handle additional cases.

**Comment by jakeleventhal** [comment]
not stale
